main/src/main/java/com/epam/mentoring/webservices/dao/UserDefaultDAO.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.mentoring.webservices.dao;
+
+import java.sql.SQLException;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+import com.epam.mentoring.webservices.bean.User;
+import com.epam.mentoring.webservices.exception.FileUploadException;
+
+public class UserDefaultDAO extends BeanDAO<User> implements UserDAO {
```

Comment:
In the next (hibernate\JPA) task , please try to use declarative transactions instead of bean management transactions.